pkgs/games/devilutionx/default.nix

```diff
@@ -1,20 +1,26 @@
 { lib, stdenv, fetchFromGitHub, cmake, SDL2, SDL2_mixer, SDL2_ttf, libsodium, pkg-config }:
 stdenv.mkDerivation rec {
-  version = "1.0.1";
-  pname = "devilutionx";
+  version = "2020-10-20";
+  pname = "devilutionx-unstable";
```

Comment:
The pname should not change here. unstable should be part of the version if we only have one package for devolutionx.